client/adventureMap/mapHandler.cpp

```diff
@@ -925,7 +925,7 @@ void CMapHandler::CMapBlitter::blit(SDL_Surface * targetSurf, const MapDrawingIn
 		{
 			for (realPos.y = initPos.y, pos.y = topTile.y; pos.y < topTile.y + tileCount.y; pos.y++, realPos.y += tileSize)
 			{
-				const int3 color(0x555555, 0x555555, 0x555555);
+				const int3 color(0x55, 0x55, 0x55);

```

Comment:
Please use ColorRGBA instead. This is not position, but color. No need to use int3 here

Reply:
Done.